kanidmd/idm/src/server.rs

```diff
@@ -929,10 +937,17 @@ impl<'a> QueryServerTransaction<'a> for QueryServerWriteTransaction<'a> {
         self.d_info.d_uuid
     }
 
+    /// Gets the in-memory domain_name element
     fn get_domain_name(&self) -> &str {
         &self.d_info.d_name
     }
 
+    fn get_domain_display_name(&self) -> String {
+        // TODO: lol this is probably terrible
+        String::from("filler text from server.rs qst qstwt")
```

Comment:
That's fine, you just need the cowcell later that does the reload on change thingo.

Reply:
🐄 🤠